identity-account/Cargo.toml

```diff
@@ -36,15 +39,30 @@ features = ["blake2b", "ed25519", "hmac", "pbkdf", "sha", "slip10", "std"]
 [dependencies.iota_stronghold]
 git = "https://github.com/iotaledger/stronghold.rs"
 rev = "6dd92dc9743eba2f3b4126425e7572470d92c80b"
+optional = true
 
 [dependencies.stronghold_engine]
 git = "https://github.com/iotaledger/stronghold.rs"
 rev = "6dd92dc9743eba2f3b4126425e7572470d92c80b"
+optional = true
 
 [dev-dependencies]
 rand = { version = "0.8" }
 rusty-fork = { version = "0.3" }
-tokio = { version = "1.5", features = ["macros", "rt", "rt-multi-thread", "sync"] }
+tokio = { version = "1.7", features = [
+    "macros",
+    "rt",
+    "rt-multi-thread",
+    "sync"
+] }
 
 [features]
 mem-client = []
+stronghold = [
+    "iota_stronghold",
+    "stronghold_engine",
+    "riker",
+    "tokio/rt-multi-thread",
+]
+wasm = ["identity-iota/wasm"]
+default = ["stronghold", "identity-iota/async"]
```

Comment:
better to have an `async` feature that enables this feature in `identity-iota` (`default = ["stronghold", "async"]`). at the moment it's a bit awkward to disable stronghold and enable the `async` feature